src/Database/Type.php

```diff
@@ -21,7 +21,7 @@
  * Encapsulates all conversion functions for values coming from database into PHP and
  * going from PHP into database.
  */
-class Type
+class Type implements TypeInterface
```

Comment:
Right, I understand now. I'll update the pull request implementing the interface into the type classes then.